pkg/auto/exporthttp/job/job.go

```diff
@@ -28,15 +29,22 @@ var (
 	errNoSensorsRetrieved = errors.New("no sensors retrieved")
 )
 
+// boltKeyTemplate is the template string used to generate the bolt key
+// "${AutoName}_${config.Root.Name}_${Job.Name}"
+// It is used to store the job's last previous execution timestamp in the database.
+const boltKeyTemplate = "%s_%s_%s"
+
 const defaultTimeout = time.Second * 5
 
 // Job represents an exporthttp automation task that executes Do to send a POST request
 type Job interface {
-	GetName() string
 	GetUrl() string
 	GetSite() string
-	GetExecutionAfter(t time.Time) <-chan time.Time
+	GetName() string
+
+	GetExecutionAfter() <-chan time.Time
 	SetPreviousExecution(t time.Time)
+	GetSchedule() *jsontypes.Schedule
```

Comment:
I think this method can also be removed, it's only ever used from within other methods on Job